modules/resources/templates/training-cti.html

```diff
@@ -9,276 +10,130 @@
+            <ul>
+              {% for training in parsed.cti_training.values() %}
+              <li>
+                <a href="#{{training.id}}">{{training.short_name}}</a>
+              </li>
+              {% endfor %}
+            </ul>
+          </div>
+        </div>
+        <div class="getting-started-content">
+          <h1>CTI Trainings</h1>
+          <p>
+            The goal of this training is for students to understand the following:
+          </p>
+          <ul>
+            <li>
+              What ATT&CK is and why it’s useful for cyber threat intelligence
+              (CTI)
+            </li>
+            <li>
+              How to map to ATT&CK from both finished reporting and raw data
+            </li>
+            <li>
+              Why it’s challenging to store ATT&CK-mapped data and what you should
+              consider when doing that
+            </li>
+            <li>How to perform CTI analysis using ATT&CK-mapped data</li>
+            <li>How to make defensive recommendations based on CTI analysis</li>
+          </ul>
+          <p>
+            The modules below come with exercises intended to increase and test
+            your understanding of ATT&CK techniques and applications. It is
+            specified when are unguided (not walked through in the video or
+            reports).
+          </p>
+          <h2>Modules</h2>
+          {% for training in parsed.cti_training.values() %}
+          <div class="anchor" id="{{training.id}}"></div>
+          <div class="card w-60 card-training">
+            <div class="card-body training-card-body">
+              <div class="training-header">
+                <div class="training-text-container">
+                  <strong class="title">{{ training.name }}</strong>
+                  <div class="date">{{ training.module }}</div>
+                </div>
+                <p></p>
+                <div class="material">
+                  <a href="{{ training.video }}">
+                    <button type="button" class="slide-button">
+                      <span class="slide-button-text">
+                        <span> Video </span>
+                      </span>
+                      <span class="slide-button-icon">
+                        <i class="fa fa-external-link"></i>
+                      </span>
+                    </button>
+                  </a>
+                  <a href="{{ training.slides }}">
+                    <button type="button" class="slide-button slide-button-secondary">
+                      <span class="slide-button-text">
+                        <span> Slides </span>
+                      </span>
+                      <span class="slide-button-icon">
+                        <i class="fa fa-external-link"></i>
+                      </span>
+                    </button>
+                  </a>
                 </div>
-            </div> <!-- end use cases -->
+              </div>
+              <p></p>
```

Comment:
It looks like this empty paragraph tag may be placed here for the additional margin it provides; if so, in lieu of the paragraph tag, I would recommend wrapping the exercises below the training headers (lines 101-124) in a `div` with the `mt-3` class. This will provide spacing between the header & the exercises and also remove the additional spacing at the bottom of Module 1:
<img width="841" alt="image" src="https://github.com/mitre-attack/attack-website/assets/48832936/72802ac9-ddfa-489a-aa0c-8d2dcb359402">